godot-core/src/builtin/collections/packed_array.rs

```diff
@@ -378,6 +378,27 @@ macro_rules! impl_packed_array {
             pub fn as_inner(&self) -> inner::$Inner<'_> {
                 inner::$Inner::from_outer(self)
             }
+
+            /// Create array filled with default elements.
+            fn default_with_size(n: usize) -> Self {
+                let mut ret = Self::new();
+                ret.resize(n);
+                ret
+            }
```

Comment:
Could you rename `ret` to `array`, as well?

Reply:
Done